esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -1641,8 +1662,19 @@ bool CWsESDLConfigEx::onListESDLBindings(IEspContext &context, IEspListESDLBindi
             bool processCreated = false;
             if (!lastBinding || (strcmp(lastBinding->getEspProcess(), binding->getEspProcess()) != 0))
             {
+                const char* curProc = binding->getEspProcess();
+                for ( ; procNameInd < allProcNames.length() && strcmp(allProcNames.item(procNameInd), curProc) < 0; procNameInd++)
```

Comment:
I can see this portion of the comparison getting misinterpreted and wrongfully corrected to strcmp != 0
"strcmp(allProcNames.item(procNameInd), curProc) < 0"

Either way it took me a while to understand what was being done here. A comment here would go a long way